packages/react/src/EditorContent.tsx

```diff
@@ -75,9 +75,7 @@ export class PureEditorContent extends React.Component<EditorContentProps, Edito
     // lifecycle methods, and React doesn't allow calling flushSync from inside
     // a lifecycle method.
     if (this.initialized) {
-      queueMicrotask(() => {
```

Comment:
Does this still prevent the initial issue we had before all those changes regarding the cursor position / rendering?

Our Netlify build didn't work for this so I can't test it right now. I'll check it out on my machine in the meantime to test your changes.